blocks/editable/format-toolbar.js

```diff
@@ -141,12 +145,14 @@ class FormatToolbar extends Component {
 			} );
 		}
 
+		const isEditingLink = this.state.isEditingLink || this.state.linkValue === '#';
```

Comment:
Hm, this like is not needed anymore, but it doesn't fix the issue for me. The input field still won't autofocus.